docs/calico-with-docker/docker-network-plugin/ManualSetup.md

```diff
@@ -37,13 +37,13 @@ up a cluster.
 > NOTE: If you are running etcd with SSL/TLS, see the (Etcd Secure Cluster)[../../EtcdSecureCluster.md]
 > documentation.
 
-### Docker
+### Docker (with multi-host networking)
 
 Follow the instructions for installing [Docker][docker].  A version of 1.9 or
 greater is required.
 
-To use the multi-host native networking feature of Docker, the Docker daemon
-needs to be run specifying a cluster store.  If using etcd as a cluster store,
+To use Calico as a Docker network plugin, the Docker daemon
```

Comment:
Normally I would agree with @matthewdupre  - I'm a fan of cutting down the redundant information from our docs. But this _is_ the docker-libnetwork guide. Its the only place I can understand going a little verbose on the docker-daemon configuration instructions.